contracts/impls/trustable/WitnetRequestBoardTrustableDefault.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./WitnetRequestBoardTrustableBase.sol";
+import "../../patterns/Destructible.sol";
+
+/// @title Witnet Request Board "trustable" implementation contract.
+/// @notice Contract to bridge requests to Witnet Decentralized Oracle Network.
+/// @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
+/// The result of the requests will be posted back to this contract by the bridge nodes too.
+/// @author The Witnet Foundation
+contract WitnetRequestBoardTrustableDefault
+    is 
+        Destructible,
+        WitnetRequestBoardTrustableBase
+{  
+    constructor(bool _upgradable, bytes32 _versionTag)
+        WitnetRequestBoardTrustableBase(_upgradable, _versionTag, address(0))
+    {}
+
+
+    // ================================================================================================================
+    // --- Overrides 'Destructible' -----------------------------------------------------------------------------------
+
+    /// Destroys current instance. Only callable by the owner.
+    function destroy() external override onlyOwner {
+        selfdestruct(payable(msg.sender));
+    }
+
+
+    // ================================================================================================================
+    // --- Overrides 'Payable' ----------------------------------------------------------------------------------------
+
+    /// Gets balance of given address.
+    function balanceOf(address _from)
```

Comment:
must this function be in this contract? I've seen it's only use in the WRBOMGX and maybe it can be defined there